pkg/backingstore/backingstore.go

```diff
@@ -824,8 +821,8 @@ func RunReconcile(cmd *cobra.Command, args []string) {
 	}
 	backingStoreName := args[0]
 	klient := util.KubeClient()
-	intervalDuration := time.Duration(3)
-	util.Panic(wait.PollImmediateInfinite(intervalDuration*time.Second, func() (bool, error) {
+	intervalSec := time.Duration(3)
```

Comment:
Referring to Guy's suggestion on the previous PR:
https://github.com/noobaa/noobaa-operator/pull/646#discussion_r651707785

Reply:
Just left this change alone, unrelated to the issue. Reverted to the original.